### PR TITLE
Rs 2022 06 26

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 4/04_00_ARC04_6.txt [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+"Yeah, this is a pretty nice room.
 }
 [sha:1e9f4ce6657727267039b5841960a7963365c122]{
 -- Page 10016, Offset 6574.
 -- これなら一晩過ごしても文句ないかな」
-I wouldn't mind spending an entire night in here."
+I don't think anyone could complain about spending a night in here."
 }
 [sha:3a80f7e7b0201ba13a63987d5157c203237f93e3]{
 -- Page 365, Offset 6575.
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6.txt:12]: 6577: I'm left speechless by this turn of events

File: script/Arcueid/Day 4/04_00_ARC04_6.txt [hunk 2]
```diff
@@ -31,12 +31,13 @@ I'm speechless at this turn of events.
...
 }
 [sha:d08107031da1d8dc0be16c377a9bfe2c0696c73b]{
 -- Page 366, Offset 6580.
 -- わたしの部屋は守りには向かないから……そうね、見晴らしのいい、高いところに行きましょうか」
-My room is not suitable for protection after all... That's right, we should go to a higher place with a better view."
+Let's see, my room probably isn't the best place to hole up...
+ Aha, how about we head somewhere that's higher up and has a better view?"
 }
 [sha:7cd0c5538b14d5cb9e6e53caea1e29eb1affd53b]{
 -- Page 367, Offset 6581.
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6.txt:43]: offset 6582: no comma after street

File: script/Arcueid/Day 4/04_00_ARC04_6.txt [hunk 3]
```diff
@@ -66,17 +69,24 @@ Human architecture really is advancing at a remarkable pace.
...
 -- Page 368, Offset 6587.
 -- ま、どっちでもいっか！　ほら、見て見て志貴、さっきまでいた路地裏があんなにちっちゃくなってる！」
-Well, it doesn't really matter! Look over there, Shiki, the alleyway we were in just now has become so tiny!"
+Well, it doesn't really matter! Ooh, ooh, over there, Shiki! The alleyway we were just in is so tiny now!"
 }
 [sha:e475d6c2e3c957438197e961490d17ca8ffb6245]{
 -- Page 369, Offset 6588.
 -- 閉め切られたカーテンをわずかに開けて、アルクェイドは窓から見下ろす風景を楽しんでいる。
-With the drawn curtains slightly open, Arcueid is enjoying the view from the window.
+Arcueid has opened a small gap for herself in the curtains,
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6.txt:88]: drop comma

File: script/Arcueid/Day 4/04_00_ARC04_6.txt [hunk 4]
```diff
@@ -306,7 +328,7 @@ At the end of the day, what are you really trying to accomplish?"
 [sha:2d9cf9174b3254488730aa26d572440469d95bc4]{
 -- Page 387, Offset 6634.
 -- 「わたし？　わたしは吸血鬼を追ってるだけよ。
-"Me? I'm just trying to go after a vampire.
+"Me? I'm just chasing after that vampire.
 }
 [sha:d9fea337c853b75f1b597c1fcd36e7c2fe97bf9d]{
 -- Page 387, Offset 6635.
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6.txt:334]: 6635: hmm. I wonder if this is something we might want to take more care with. Weapon-mode Arcueid might be using 吸血鬼殺し quite specifically given its in quotes.
Maybe like "'Vampire Hunter' is my [role/job title/etc.]" or something

File: script/Arcueid/Day 4/04_00_ARC04_6.txt [hunk 5]
```diff
@@ -482,47 +512,54 @@ Okay, I'll have to make sure to remember that.
...
 -- わたしね、志貴。あなたに殺されて復元するまでのあいだ、あなたは教会の人間だと思っていたの。
-You know, Shiki. From when I was killed by you up until the time I restored myself, I thought that you were a member of the %{i}Church%{/i}.
+You know what, Shiki?
+ I spent the entire time between when you killed me and when I revived under the impression that you were a member of the %{i}Church%{/i}.
 }
 [sha:ac1bda3e23e0fb30c4c98fae76d128cf080512c2]{
 -- Page 400, Offset 6672.
 -- それなら敵の居場所を掴んでいる可能性もあるし、戦力としても期待できるなって」
-If that were the case, there was a chance that you knew the location of the enemy and could be counted on as a force to be reckoned with."
+If you had been,
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6.txt:531]: "had" in italics

File: script/Arcueid/Day 4/04_00_ARC04_6.txt [hunk 6]
```diff
@@ -672,12 +710,12 @@ I stare at Arcueid's face.
...
 -- Page 413, Offset 6707.
 -- ……ふくよかな胸が上下しているところを見ると、呼吸はしているようだ。
-Observing her well-rounded chest as it rises and falls, it would appear that she is breathing.
+Judging by the gentle rise and fall of her well-rounded chest, it seems she's still breathing.
 }
 [sha:c176751911a02f8761552fe69dfb1353b8f41d97]{
 -- Page 413, Offset 6708.
 -- ベッドに横たわった体はぴくりとも動かない。
-The body lying on the bed does not move a muscle.
+She lies on the bed, her body perfectly still.
```
Comment (Avuxo) [inline script/Arcueid/Day 4/04_00_ARC04_6.txt:718]: offset 6715: Demure, yet sensual

**Review by Avuxo — APPROVED** [review]
what dick opened three bottles of wine and just left them out
image

**Comment by rschlaikjer** [comment]
> what dick opened three bottles of wine and just left them out
Arc knows you gotta let it breathe

File: script/Arcueid/Day 4/04_00_ARC04_6.txt (hunk 4 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 4/04_00_ARC04_6.txt:334]: Judgement seems to be it's ok as is

**Pull request merged**